src/commcare_cloud/ansible/restart_webworker_services.yml

```diff
@@ -1,4 +1,4 @@
-- name: Rolling restart webworker services - {{ webworker_hosts }}
+- name: Rolling restart webworker services - ({{ webworker_hosts }})
```

Comment:
I'm not sure I understand this change and how it relates to the commit message

Reply:
I thought that adding the brackets would make ansible make interpret it as a variable which it did not. And after reading your last comment I understand why.